src/foam/nanos/controller/AppStyles.js

```diff
@@ -255,5 +262,31 @@ foam.CLASS({
       scroll-behavior: auto !important;
     }
   }
+
+
+  .large-subtitle {
+    font-size: 2.4rem;
+    line-height: 1.67;
+  }
+
+  .large-title {
+    text-align: center;
+    font-weight: bold;
+  }
+  @media only screen and (max-width: 320px) {
+    :root {
+      font-size: 50%;
+    }
+    .large-title { font-size: 2.8rem; }
+  }
+  @media only screen and (min-width: 321px) and (max-width: 767px) {
```

Comment:
Don't need to specify the max-width explicitly since it would be handled by the next break point anyway